packages/js/src/introductions/initialize.js

```diff
@@ -16,6 +16,8 @@ domReady( () => {
 		return;
 	}
 
+	const abortDisplay = location.href.indexOf( "page=wpseo_dashboard#/first-time-configuration" ) !== -1;
```

Comment:
So if this is true, why not:

* stop the whole JS side integration here
* do the seen `false` request here

This would leave it out of the IntroductionProvider

Reply:
I thought about that and it seemed to me as more future-proof.

Basically, the fetch is also dependent on the modal component that's about to be shown. And in the `IntroductionProvider`, we touch that via `registerComponent` as well. So, I figured to have the fetch as close to the last place we touch the about-to-be-rendered component as possible.

Reply:
> It just feels so strange. Perhaps expose window.YoastSEO._registerIntroductionComponent to just do a request to unsee that introduction instead?

What is strange? In my mind, enhancing the introduction provider to accept aborting functionality kind of makes sense and allows for other cases in the future